keyvault_create.yml

```diff
@@ -18,7 +18,11 @@
   vars:
     resource_group: "{{ resource_group_name }}"
     location: eastus
-    keyvault_name: keyvault{{ rpfx }}    
+    keyvault_name: keyvault{{ rpfx }}
+  
+  roles:
+    - Azure.azure_preview_modules
```

Comment:
if it's available in >=2.5 why to use role for testing?

Reply:
some bug not fixing  even in latest  ansible version 2.6.

Reply:
as you know, keyvault bug fixing pr in upstream is pending review for long time

Reply:
ok